### PR TITLE
Api add social media sites to admin

### PR DESCRIPTION
**Pull request by secondaryfun**
- Related Issue (include '#'):  634

- Description of changes made:

Added SocialMediaType to admin console.

- Is the feature complete/bug resolved/etc..:

Using django templates, no issues.

- Any known bugs/strange behavior:

works fine. 

- Is there specific feedback you would like on these changes:

- Screenshot(s):
image

### CODE REVIEW

**Comment by 0xZakk** [comment]
@secondaryfun be sure to make a link to the issue, not just the issue number

File: client/src/pages/MemberProfilePage/MemberProfilePage.js [hunk 1]
```diff
@@ -169,18 +167,21 @@ class MemberProfilePage extends Component {
...
+                      type="default"
+                      first_name={this.state.protectedProfileData.first_name} last_name={this.state.protectedProfileData.last_name}
+                      // name={`${this.props.profile.firstname} ${this.props.profile.lastname}` }
+                      bio={this.state.protectedProfileData.bio}
+                      hashtags={people[0].hashtags}
+                      isCurrentUser={true}
+                      onClick={console.log("Hello")}
+                      instagram={people[0].instagram}
+                      pinterest={people[0].pinterest}
+                      facebook={people[0].facebook}
```
Comment (0xZakk) [inline client/src/pages/MemberProfilePage/MemberProfilePage.js:182]: These links aren't working for some reason. The URL it creates is like `http://localhost:3000/www.instagram.com`

**Review by 0xZakk — CHANGES_REQUESTED** [review]

File: client/src/pages/MemberProfilePage/MemberProfilePage.js (hunk 1 above)
Comment (0xZakk) [reply client/src/pages/MemberProfilePage/MemberProfilePage.js:182]: Also looks like it doesn't include the user's social media account. We only want to display that if they have that account on CoTripper and we want them to be able to add that account

**Comment by secondaryfun** [comment]
Apologies, this branch received the wrong commit.  Closing to resubmit.

**Pull request closed without merging**